libres/lib/enkf/row_scaling.cpp

```diff
@@ -15,19 +15,18 @@
    See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
    for more details.
 */
+#include "ert/python.hpp"
```

Comment:
Consider moving to where the other stuff from `ert/` is included, and also consider using `<ert/python.hpp>`.